src/page-modules/assistant/server/journey-planner/journey-gql/via-trip.gql

```diff
@@ -0,0 +1,116 @@
+query ViaTrips(
+  $from: Location!
+  $to: Location!
+  $when: DateTime
+  $via: [ViaLocationInput!]!
+) {
+  viaTrip(
+    from: $from
+    to: $to
+    searchWindow: "PT2H" # 9 hours
+    dateTime: $when
+    via: $via
+  ) {
+    ...viaTrip
+  }
+}
+
+fragment viaTrip on ViaTrip {
+  routingErrors {
+    ...routingError
+  }
```

Comment:
Since we are not using this data it can be removed. Not quite sure how this data works and it might be something that we should look into, but we should add it when we need the data

Reply:
Removed👍